stylesheets/components/buttons.css

```diff
@@ -0,0 +1,183 @@
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.buttons{
```

Comment:
the name could have been different. I assume you are using this on a div. If thats the case give it a generic name as component-wrapper and use it across the project

Reply:
Ya, that was just to visualize, I didn't need it in my library I only need the buttons classes here, thanks for pointing it out, I will remove it.